docs/source/guide/manage_artifacts/index.rst

```diff
@@ -44,3 +44,4 @@ imagine how difficult it would be to maintain correlations between the two. Line
 
    artifact_reuse
    storage_location/index
+   storage_backend/index
```

Comment:
I am not sure if the distinction between "storage location" and "storage backend" will be clear to our user. As such, why not combine these two sections? That is, the current PR's contents relating to MLflow can be put under existing `Changing Storage Location` section and be titled "Storing model artifact values in MLflow", i.e.:

```
Changing Storage Location
  - Storing Artifact Metadata in PostgreSQL
  - Storing Artifact Values in Amazon S3
  - Storing Model Artifact Values in MLflow
```

Reply:
I understand why you have this comment. However, location and backend are at different levels of abstraction. I think putting them together is more confusing.

MLflow can have its own storage location, it can be local/s3/postgres. In this case, we are basically saying for this type of artifact(ML model), we use MLflow to handle the storage; it could be s3/local/gcp/..., and we don't care how MLflow is configured, we just need to specify the host of MLflow. However, for LineaPy itself, we are the host of LineaPy. We need to configure the underlying storage location and how the catalog(db) is hosted.

Reply:
is the name storage_location and storage_backend also used by mlflow for those two configs? if so, then it is probably already clear for mlflow users as you said.

Reply:
hmm i think they have a backend store and artifact store. the backend store is configured using --backend-store-uri and the artifact store is configured using --default-artifact-root.

not sure if i'm completely right here but the storage_location here will be the --backend-store-uri of mlflow and storate_backend will be mlflow?

Reply:
@mingjerli Ok, here's my understanding based on your explanation above:

![image](https://user-images.githubusercontent.com/13392380/199847210-e39b145c-a5c1-4e99-b8f2-f05423f3d549.png)

If this understanding is correct, then I suggest we make things more clear in the landing page of `Changing Storage Backend` section, like so:

> Out of the box, LineaPy is the default storage backend for all artifacts. For certain storage backends in use (e.g., storing model artifacts in MLflow), saving one more copy of the same artifact into LineaPy causes sync issue between the two systems. Thus, LineaPy supports using different storage backends for certain data types (e.g., ML models). This support is essential for users to leverage functionalities from both LineaPy and other familiar toolkit (e.g., MLflow).
> 
> *NOTE: Storage backend refers to the overall system handling storage and should be distinguished from specific storage locations such as Amazon S3. For instance, LineaPy is a storage backend that can use different storage locations.*
> 
> - Using MLflow as Storage Backend to Save ML Models

Reply:
@lionsardesai I think what you are referring to is how to setup/run an MLflow server, not how end users connect to MLflow. End users are using [`mlflow.set_tracking_uri`](https://www.mlflow.org/docs/latest/python_api/mlflow.html#mlflow.set_tracking_uri) and [`mlflow.set_registry_uri`](https://www.mlflow.org/docs/latest/python_api/mlflow.html#mlflow.set_registry_uri) for configuration to connect to MLflow. They don't need to know how the MLflow server has been set up exactly(as long as their IT/ops people tell them which tracking_uri and registry_uri they should use). And the crazy part is you can almost put anything into `set_tracking_uri`, filepath, s3path, database ...

@yoonspark agree, added